Ports/gzip/package.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env -S bash ../.port_include.sh
+port=gzip
+version=1.12
+useconfigure="true"
+use_fresh_config_sub="true"
+workdir="gzip-${version}"
+config_sub_paths=("build-aux/config.sub")
+files="https://ftp.gnu.org/gnu/gzip/gzip-${version}.tar.gz gzip-${version}.tar.gz
+https://ftp.gnu.org/gnu/gzip/gzip-${version}.tar.gz.sig gzip-${version}.tar.gz.sig
+https://ftpmirror.gnu.org/gnu/gnu-keyring.gpg gnu-keyring.gpg"
+auth_type="sig"
```

Comment:
```suggestion
auth_type='sha256'
```

Signature-based authentication is a bit flawed, especially if you aren't checking the keyring that you are downloading from the server. I'd strongly prefer using normal sha256sums for checking downloads.